README.md

```diff
@@ -5,20 +5,34 @@ This extension contains various components of the [Rust](https://www.rust-lang.o
 
 ## Mold
 
-In order to use the (fast) [mold](https://github.com/rui314/mold) linker:
+In order to use the (fast) [`mold`](https://github.com/rui314/mold) linker:
 
-1. Add `org.freedesktop.Sdk.Extension.llvm13` along with this extension in order to get clang.
+1. Add `org.freedesktop.Sdk.Extension.llvm13` along with this extension in order to get `clang`.
 2. Add `/usr/lib/sdk/llvm13/bin` to `append-path`. See [llvm13 SDK extension readme](https://github.com/flathub/org.freedesktop.Sdk.Extension.llvm13) for more information.
-3. Add `.cargo/config.toml` in the root of the repository with the following content:
-
-```toml
-[target.x86_64-unknown-linux-gnu]
-linker = "clang"
-rustflags = ["-C", "link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"]
+3. Set environment variables:
+    - `CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER` to `clang`, and
+    - `CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS` to `-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold`.
+
+In total, the changed parts of your flatpak manifest should look like this:
+
+```json
+{
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm13"
+    ],
+    "build-options": {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm13/bin",
+        "env": {
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER": "clang",
```

Comment:
can we do set the same env variables for aarch64?

Reply:
Yeah, we probably should (not that I can really test if it would work)

Reply:
I've added aarch64